DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -313,14 +319,20 @@ private function loadSymfony(ContainerBuilder $container, XmlFileLoader $loader,
         }
     }
 
+    /**
+     * @param ContainerBuilder $container
+     * @param XmlFileLoader    $loader
+     * @param array            $config    Configuration section for the tags node
+     * @param string|bool      $client    Name of the client or false when a custom client is used
```

Comment:
`string|false`. But I don't like multi-type params. So I'd use the empty string instead of false.